validator/server_api/validation_api.go

```diff
@@ -122,6 +126,27 @@ func (a *ExecServerAPI) removeOldRuns(ctx context.Context) time.Duration {
 func (a *ExecServerAPI) Start(ctx_in context.Context) {
 	a.StopWaiter.Start(ctx_in, a)
 	a.CallIteratively(a.removeOldRuns)
+	if a.consumer != nil {
```

Comment:
We want multiple (hot-configurale number) of threads consuming and working on messages concurrently

Reply:
Changed to multiple consumers.

As for hot-configurable, you mean we need to have ability to stop some/ start more consumers on the fly without restart ?